src/actions/bitcoinActions.js

```diff
@@ -116,14 +117,22 @@ export const loadBitcoinAddresses = () => {
 
     const loaded: string[] = Object.keys(keys);
 
-    dispatch(setBitcoinAddresses(loaded));
+    if (loaded.length) {
+      dispatch(setBitcoinAddresses(loaded));
+    }
   };
 };
 
-const fetchBalanceAction = async (address: string): Promise<BitcoinReducerAction> => {
-  const unspentTransactions = await getAddressUtxos(address);
+const outdatedAddresses = (addresses: BitcoinAddress[]): BitcoinAddress[] => {
+  const threshold = Date.now() - REFRESH_THRESHOLD;
```

Comment:
Why do you need the `REFRESH_THRESHOLD` param? Maybe it would be worth mentioning this in the comment

Reply:
If we need a comment for such a simple line, something's not fine. Maybe those are really bad names?

Outdated addresses would be the addresses that haven't been updated for some time (The `REFRESH_THRESHOLD`). This specific constant will determine what's the oldest we still consider "updated", so we return only the ones that are older than that.

What do you think about looking for better wordings here? I'm open to suggestions.